### PR TITLE
build script for Crontab and spdystream packages

### PR DESCRIPTION
**Pull request by Sapana-Khemkar**

### CODE REVIEW

**Review by amitsadaphule — CHANGES_REQUESTED** [review]
The PR does not seem to be constrained to the 2 packages. Shows 88 files changed. Also, there are conflicts. Please address these issues and then we can proceed with further review.

**Comment by Sapana-Khemkar** [comment]
> The PR does not seem to be constrained to the 2 packages. Shows 88 files changed. Also, there are conflicts. Please address these issues and then we can proceed with further review.
Fixed conflict issue and now total files changed are 3.

**Comment by amitsadaphule** [comment]
The PR still seems to include the build-script for an additional package `genproto`. Is that intentional?

Also, `spdystream` is missing LICENSE file.

**Comment by Sapana-Khemkar** [comment]
> The PR still seems to include the build-script for an additional package `genproto`. Is that intentional?
> 
> Also, `spdystream` is missing LICENSE file.

My bad. By mistake I pushed new packages to this branch. reverted these changes. Also added missing LICENSE for spdystream

File: s/spdystream/spdystream-ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+wget https://golang.org/dl/go$GO_VERSION.linux-ppc64le.tar.gz  
+tar -C /bin -xzf go$GO_VERSION.linux-ppc64le.tar.gz  
+rm -f go$GO_VERSION.linux-ppc64le.tar.gz 
+
+#set GO PATH
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/home/tester/go
+
+mkdir -p $GOPATH/$PACKAGE_NAME/src
+cd $GOPATH/$PACKAGE_NAME/src
```
Comment (amitsadaphule) [inline s/spdystream/spdystream-ubi8.4.sh:43]: Should this be

```
mkdir -p $GOPATH/src/github.com/moby
cd $GOPATH/src/github.com/moby
```

to be consistent with go conventions?

File: s/spdystream/spdystream-ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+git clone https://github.com/moby/spdystream.git
+cd $PACKAGE_NAME
+git checkout $PACKAGE_COMMIT_HASH
+
+#install dependencies
+go mod init github.com/spdystream
+go mod tidy
+
+go get ./...
```
Comment (amitsadaphule) [inline s/spdystream/spdystream-ubi8.4.sh:53]: The repo already contains go.mod. So, is it really necessary to call `go mod xx` and `go get xx` commands? Can't we directly do a `go build ./...` instead?

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: s/spdystream/spdystream-ubi8.4.sh (hunk 1 above)
Comment (Amit-Sirohi) [reply s/spdystream/spdystream-ubi8.4.sh]: Changes done

**Comment by Sapana-Khemkar** [comment]
@seth-priya @kandarpamalipeddi can one of you review this PR. All review comments are already in place. I think Amit Sadaphule did not get chance to review n merge

**Pull request merged**